app/lib/frontend/static_files.dart

```diff
@@ -333,3 +350,33 @@ Future<void> updateWebCssBuild() async {
     throw Exception(message);
   }
 }
+
+@visibleForTesting
+class ParsedStaticUrl {
```

Comment:
Can this have a tiny documentation comment?

Reply:
I'm guessing it because we need to handle both URL hash and query hash, during the migration?

Reply:
Yes, after we'd no longer use the request parameter hash, this will become much simpler.